NewSpring/ApollosProject
# [DEV-3694] Add "Add your prayer" Button

## DESCRIPTION

### What does this PR do, or why is it needed?
This PR adds the "Add your prayer" button to the top of the prayer screen when you select an option from the menu. 
### How do I test this PR?
1. Start up your sim.
2. Go to the Prayer tab.
3. Tap on a menu option under the "Ask for prayer" section.
4. The "Ask for prayer" section should disappear and be replaced with an "Add your prayer" button at the top of the screen while also displaying the appropriate content for that tab. 

Before tapping the menu option:
![Screenshot 2019-04-28 17 13 01](https://user-images.githubusercontent.com/3229463/56870153-f2de0600-69d8-11e9-8885-5f43f29a1bd6.png)

After tapping the menu option:
![Screenshot 2019-04-28 17 13 07](https://user-images.githubusercontent.com/3229463/56870156-fc676e00-69d8-11e9-9eec-7954102d23f6.png)

---

> I am affirming this is my _best_ work ([Ecclesiastes 9:10](https://www.bible.com/bible/97/ECC.9.10.MSG))

## TODO

- [x] Closes [DEV-3694](https://newspring.atlassian.net/browse/DEV-3694)

## REVIEW

**Manual QA**


**Code Review: Questions to consider**


---

> The purpose of PR Review is to _improve the quality of the software._
